tc-build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname $0)/tc-vars.sh
+
+build_gpu=no
+build_arm=no
+
+if [ "$1" = "--gpu" ]; then
```

Comment:
I don't think it's a big deal, but it might be nicer to stick this in a shift loop and read off multiple parameters, for the future situation that we support more than 1, but also so that you can error out if someone specifies both --gpu and --arm, rather than respecting whichever is first listed.